trackpy/feature.py

```diff
@@ -521,6 +524,14 @@ def batch(frames, diameter, minmass=100, maxsize=None, separation=None,
         If specified, information relevant to reproducing this batch is saved
         as a YAML file, a plain-text machine- and human-readable format.
         By default, this is None, and no file is saved.
+    threads : integer, optional
+        If 0, no threading is used. An integer greater 0 is treated as the
+        number of threads used to process frames in parallel. If `threads` is
+        None then the number returned by ``os.cpu_count()`` is used.
```

Comment:
FYI I tested, seems to be only slightly faster than `os.cpu_count() - 1`. But this seems the most sensible default.

Reply:
Yeah, I just mirrored the behavior of `Pool`'s input argument `processes`.

Reply:
I was thinking about this. For me, `threads=None` would logically be "no threading", and `threads=0` could be the `os.cpu_count()`.

In either case, please keep the default behaviour "no threading". We do not want to surprise our users too much on a minor release (`v0.5` is not coming soon).